packages/components/src/components/Timerange/Timerange.tsx

```diff
@@ -570,24 +581,24 @@ const Calendar = styled.div`
 
     .rdrStartEdge,
     .rdrEndEdge {
-        background: ${({ theme }) => theme.BG_GREEN};
-        color: ${({ theme }) => theme.TYPE_WHITE};
-        box-shadow: ${({ theme }) => theme.BG_GREEN} 0px 0px 0px 2px;
-        border-radius: 4px;
+        background: ${({ theme }) => theme.backgroundPrimaryDefault};
+        color: ${({ theme }) => theme.textOnPrimary};
+        box-shadow: ${boxShadows.focusedLight};
+        border-radius: ${borders.radii.xs};
         z-index: ${variables.Z_INDEX.BASE};
```

Comment:
Shouldn't we also create design tokens for z-indexes? Maybe just rename it to have it without upper case

Reply:
Oh! I found `ZIndices`. Not ready yet, OK.

TIL: I didn't know that index has plural indices!

Reply:
Nice :D Im thinking, for now we can copy values from `variables.Z_INDEX` to `ZIndices` as well for future use. also – easier to mass replace later.

Reply:
Not necessarily as a part of this PR

Reply:
New PR with z-indices here: https://github.com/trezor/trezor-suite/pull/9702